.eslintrc.json

```diff
@@ -50,7 +66,10 @@
       "@typescript-eslint/semi": ["error", "never"],
       "@typescript-eslint/type-annotation-spacing": "error",
       "@typescript-eslint/unbound-method": "error",
-      "no-console": "off"
+      "no-console": "off",
+      "import/no-unresolved": "off",
```

Comment:
do we need these extra rules?

Reply:
The extra rules were getting applied as we switched from github/es6 to github/recommended. The es6 config is not supported in the latest version and has been merged in recommended. 
So have just marked the extra rules off as they were breaking the lint.